mokapot/mokapot.py

```diff
@@ -82,13 +83,24 @@ def main():
                 dataset.add_proteins(proteins)
 
     # Define a model:
-    model = PercolatorModel(
-        train_fdr=config.train_fdr,
-        max_iter=config.max_iter,
-        direction=config.direction,
-        override=config.override,
-        subset_max_train=config.subset_max_train,
-    )
+    if config.init_weights:
+        model_files = os.listdir(str(config.init_weights))
+        if len(model_files) != config.folds:
+            raise RuntimeError(
+                "Number of loaded models should be equal to the number of folds."
+            )
```

Comment:
I think it makes more sense to perform this test in the `brew()` function, so that we can catch errors when folks use the Python API as well.